src/Components/Search/NewSearch/Mobile/SearchResultsList.tsx

```diff
@@ -32,9 +38,27 @@ const SearchResultsList: FC<SearchResultsListProps> = ({
   relay,
   viewer,
   query,
+  selectedPill,
+  fetchCounter,
   onClose,
 }) => {
+  const tracking = useTracking()
   const options = extractNodes(viewer.searchConnection)
+
+  useEffect(() => {
+    if (!relay.isLoading()) {
+      tracking.trackEvent({
+        action_type:
+          options.length > 0
+            ? DeprecatedSchema.ActionType.SearchedAutosuggestWithResults
+            : DeprecatedSchema.ActionType.SearchedAutosuggestWithoutResults,
+        context_module: selectedPill.analyticsContextModule,
+        query: query,
+      })
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
same explanation as [here](https://github.com/artsy/force/pull/12567/files#r1245177229), I don't want to execute this effect function on query or selectedPill change, only manually when updating fetchCounter